routing/pathfind.go

```diff
@@ -1099,6 +1100,182 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 	return pathEdges, distance[source].probability, nil
 }
 
+// blindedPathRestrictions are a set of constraints to adhere to when
+// choosing a set of blinded paths to this node.
+type blindedPathRestrictions struct {
+	// minNumHops is the minimum number of hops to include in a blinded
+	// path. This doesn't include our node, so if the minimum is 1, then
+	// the path will contain at minimum our node along with an introduction
+	// node hop. A minimum of 0 will include paths where this node is the
+	// introduction node and so should be used with caution.
+	minNumHops uint8
```

Comment:
yeah - If we have no nodes around us supporting route blinding, then we can have minNumHops be 0 here which would essentially return a path with just our node & then we can pad it with dummy hops. 

It would essentially just allow us to have our node (the destination node) also be the introduction node